src/reference/glossary.rst

```diff
@@ -60,6 +60,12 @@ Glossary
 
       Some :term:`workflows<workflow>` produce a single, synonymous dataset, like Zika. Others, like seasonal flu, produce many datasets.
 
+   narrative
+
+      A Markdown file with additional syntax to specify a :term:`dataset` to display alongside the text.
+
+      Viewable on nextstrain.org or with :term:`Auspice` via the :doc:`cli:commands/view` or :doc:`auspice view <auspice:introduction/how-to-run>` commands.
+
```

Comment:
Link out to existing docs?

- https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html
- https://docs.nextstrain.org/en/latest/tutorials/narratives-how-to-write.html